mantidimaging/gui/windows/live_viewer/model.py

```diff
@@ -217,15 +221,22 @@ def sort_images_by_modified_time(images: list[Image_Data]) -> list[Image_Data]:
         """
         return sorted(images, key=lambda x: x.image_modified_time)
 
-    def _handle_directory_change(self, directory: str) -> None:
+    def _handle_notified_of_directry_change(self, directory: str) -> None:
+        """
+        Quickly handle the notification event for a change, and start the timer if needed. The means that if files are
+        arriving faster than _handle_directory_change() can process them, we don't end up with a queue of work to do
```

Comment:
** Potential typo here, maybe change to "If files are arriving faster than _handle_directory_change() can process them, so that we don't end up with a queue of work to do"

I'd also probably add a brief description of the timers purpose here to clear any possibility of misunderstanding it's purpose.